packages/search/src/features/registration/birth/service.ts

```diff
@@ -167,12 +168,8 @@ async function indexAndSearchComposition(
 
   await createIndexBody(body, composition, authHeader, bundleEntries)
   await indexComposition(compositionId, body, client)
-  if (
-    body.type !== 'IN_PROGRESS' &&
-    body.type !== 'WAITING_VALIDATION' &&
-    body.type !== 'VALIDATED' &&
-    body.type !== 'ISSUED'
-  ) {
+
+  if (body.type === DECLARED_STATUS) {
```

Comment:
@sifulovi let's check for IN_PROGRESS as well here. We used to skip the in progress declarations previously but after we added the minimum fields required check in the duplication logic, it should be safe to check them for duplicates